constructor/winexe.py

```diff
@@ -255,6 +287,9 @@ def make_nsi(info, dir_path, extra_files=()):
                                       '${NAME} ${VERSION} (Python ${PYVERSION} ${ARCH})'
                                       )),
         ('@EXTRA_FILES@', '\n    '.join(extra_files_commands(extra_files, dir_path))),
+        ('@CUSTOM_WELCOME_FILE@', custom_nsi_insert_from_file(info.get('welcome_file', ''))),
```

Comment:
Following the comment on the template, we will need a `custom_welcome` boolean added to the `ppd` dictionary.